WeakAuras/RegionTypes/DynamicGroup.lua

```diff
@@ -536,6 +562,10 @@ local function modify(parent, region, data)
     end
   end
 
+  if WeakAuras.IsPaused() and region.suspended <= 0 then
```

Comment:
I don't understand this. Why should modify change the suspended count?

Reply:
I ended up having the region methods check WeakAuras.IsPaused() before continuing, which is perhaps a bit saner.